com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkDictionary.cs

```diff
@@ -36,25 +37,6 @@ public class NetworkDictionary<TKey, TValue> : NetworkVariableBase, IDictionary<
         /// <param name="readPerm">The read permission to use for this NetworkDictionary</param>
         public NetworkDictionary(NetworkVariableReadPermission readPerm) : base(readPerm) { }
 
-        /// <summary>
-        /// Creates a NetworkDictionary with a custom value and custom settings
-        /// </summary>
-        /// <param name="readPerm">The read permission to use for this NetworkDictionary</param>
-        /// <param name="value">The initial value to use for the NetworkDictionary</param>
-        public NetworkDictionary(NetworkVariableReadPermission readPerm, IDictionary<TKey, TValue> value) : base(readPerm)
```

Comment:
These constructors don't seem to really work.  And really, I'm trying to think how in practice you would use these - you usually declare netvars as members of your network behavior, and it's not very doable to then pre-initialize these to a dictionary.  I tried to force them to work by replacing one at runtime, but then it doesn't seem to properly dirty / update the clients.  After this failure and realizing users would stumble similarly, I decided to simply require users to instantiate the dictionary blank and then add in values through the normal interface